ray-operator/test/support/test.go

```diff
@@ -121,3 +126,34 @@ func (t *T) NewTestNamespace(options ...Option[*corev1.Namespace]) *corev1.Names
 	})
 	return namespace
 }
+
+func (t *T) StreamKubeRayOperatorLogs() {
+	ctx, cancel := context.WithCancel(context.Background())
+	t.T().Cleanup(cancel)
```

Comment:
I am not familiar with `context`. ChatGPT tells me:

> `cancel`: This is a function. Calling this function cancels the context `ctx`, releasing resources associated with it.

> `cancel` is passed as an argument to `Cleanup`, meaning that when the test finishes, the `cancel` function will be called. This effectively cancels the `ctx` context, ensuring that any operations or goroutines using `ctx` can be notified to stop and resources can be freed.

* Is the information from ChatGPT correct?
* What is the resource that this function wants to release?

Reply:
The first quote alone is quite confusing. But this
> `cancel` is passed as an argument to `Cleanup`, meaning that when the test finishes, the cancel function will be called. This effectively cancels the ctx context, ensuring that any operations or goroutines using ctx can be notified to stop and resources can be freed.

is correct. The `cancel` will terminate the stream obtained from the `stream, err := req.Stream(ctx)` and exit the loop for `scanner.Scan()` when the test is ended so that we don't leak goroutines and duplicate logs when there are multiple test cases.